src/minetest.cpp

```diff
@@ -166,6 +163,31 @@ bool Minetest::runMod(EditorState *state, const std::string &world)
 	FileFormat *writer = getFromType(FILE_FORMAT_LUA, state);
 	save_file(writer, state, mod_to + "init.lua");
 
+	// Build argument list
+	char *argv[9];
+	argv[0] = (char*)(minetest_exe).c_str();
+	argv[1] = "--worldname";
+	argv[2] = (char*)world.c_str();
+	argv[3] = "--name";
+	argv[4] = "tester";
+	argv[5] = "--address";
+	argv[6] = "";
+	argv[7] = "--go";
+	argv[8] = NULL;
```

Comment:
Will do. It was written that way as I got an error when doing initialiser lists, so split out the lines to see which one caused it, then forgot to revert.